core/src/main/java/tc/oc/pgm/events/ItemRemovedByActionEvent.java

```diff
@@ -0,0 +1,28 @@
+package tc.oc.pgm.events;
+
+import org.bukkit.entity.Item;
+import org.bukkit.event.Event;
+import org.bukkit.event.HandlerList;
+
+public class ItemRemovedByActionEvent extends Event {
+  private Item entity;
+
+  public ItemRemovedByActionEvent(Item entity) {
```

Comment:
You should not be implementing your own event for this.

Either find a method to despawn the item naturally, or just call a bukkit item despawn eveny.